### ISSUE TITLE
When can you provide the code for iOS client???

### ISSUE THREAD

**Issue by Goutham-Byteridge** [mobilequickie/amplify-passwordless-sms-auth#1]

**Comment by vajirar** [comment]
Went through the article while following the steps. Then found the empty folder. It's great to know that when we can try this out.

**Comment by emmavray** [comment]
Any update?
@mobilequickie

**Comment by Aleksansdr** [comment]
Hello @mobilequickie 
Need your update

**Comment by CameraPost** [comment]
Hi @mobilequickie, I like the idea of passwordless login by SMS. May I ask if the iOS client commit is ready? Thanks!

**Comment by Ricardo1980** [comment]
Hello @mobilequickie 
Can you tell us the main methods/apis we have to use when creating the iOS app?
Thanks a lot.

**Comment by Ricardo1980** [comment]
I have some progress creating the client, but I found a bug I cannot fix:
https://github.com/aws-amplify/aws-sdk-ios/issues/2101

What I did is trying to replicate the react app logic:

Here we see that the password is not important and that the username is the phone number.
https://github.com/mobilequickie/amplify-passwordless-sms-auth/blob/master/webClient/angular/src/app/auth.service.ts#L47

Here we see that signup is called to create the user, and after that, we have to call signIn to start the sms logic.
https://github.com/mobilequickie/amplify-passwordless-sms-auth/blob/master/webClient/angular/src/app/sign-up/sign-up.component.ts#L34

**Comment by dmennis** [comment]
@Ricardo1980  and others... I just published the iOS Sample application that works the same way as the (Angular) web client. Deploy the latest backend and then update the awsconfiguration.json in the iOS client to match your environment. 

[iOS Client](https://github.com/mobilequickie/amplify-passwordless-sms-auth/tree/master/iOSClient)

In addition, I updated and published a new version (1.14.20) of the /backend under the _amplify-passwordless-sms-auth_ Serverless Application Repository [here](https://us-west-2.console.aws.amazon.com/lambda/home?region=us-west-2#/create/app?applicationId=arn:aws:serverlessrepo:us-east-1:552623489034:applications/amplify-passwordless-sms-auth).
